libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/guidance/signboard/model/SignboardValue.kt

```diff
@@ -0,0 +1,17 @@
+package com.mapbox.navigation.ui.maps.guidance.signboard.model
+
+/**
+ * The state is returned when the signboard is ready to be rendered on the UI.
+ * @property bytes contains the signboard.
+ * @property options specifies properties of a signboard.
+ */
+class SignboardValue internal constructor(
+    val bytes: ByteArray,
+    val options: MapboxSignboardOptions
```

Comment:
Same as above re: default value `MapboxSignboardOptions.Builder().build()`

Also, can this be a `data class`? Should it be? 🤔

Reply:
Nope. As per our architecture we discussed all `Value` files are regular `class` files.

Reply:
But this is `internal` 🤔

Reply:
I think that's what we agreed to that all `Value` classes will be internal and class files.

Reply:
As this is `internal`, can we cut a ticket to revisit in the future?